src/dodal/devices/attenuator/filter_selections.py

```diff
@@ -0,0 +1,75 @@
+from ophyd_async.core import SubsetEnum
+
+
+class FilterSelection(SubsetEnum): ...
```

Comment:
I'd remove this intermediary class, and just allow FilterMotor to have any SubsetEnum, and just make sure that type is well documented in ophyd_async. That then also allows each of these filter types to be defined in ixx.py beamline module, directly alongside the device that is using them, and may allow us to move the concept of the FilterMotor out of dodal in future.